testing/src/main/java/io/stargate/it/storage/StargateContainer.java

```diff
@@ -72,7 +72,7 @@
   private static final String ARGS_PROVIDER_CLASS_NAME =
       System.getProperty("stargate.test.args.provider.class", ArgumentProviderImpl.class.getName());
   public static final File LIB_DIR = initLibDir();
-  private static final int MAX_NODES = 10;
+  private static final int MAX_NODES = 20;
```

Comment:
What's this for?

Reply:
The CI for 3.11 and 4.0 is failing, see this:
https://github.com/stargate/stargate/pull/1132#issuecomment-886561276
One of the log outputs was:
```
Step #4 - "cassandra-3.11": [ERROR]   AdditionalPortsTest Tests using too many Stargate nodes (10 maximum)
Step #4 - "cassandra-3.11": [ERROR]   AuthenticationTest Tests using too many Stargate nodes (10 maximum)
Step #4 - "cassandra-3.11": [ERROR]   JwtAuthTest Tests using too many Stargate nodes (10 maximum)
Step #4 - "cassandra-3.11": [ERROR]   PaginationTest Tests using too many Stargate nodes (10 maximum)
Step #4 - "cassandra-3.11": [ERROR]   PreparedStatementTest Tests using too many Stargate nodes (10 maximum)
```
so I was trying to fix that, but it seems that it is not an underlying issue.

Reply:
the "too many nodes" error often happens because previous tests failed and our JUnit 5 extensions did not reclaim their ports... IIRC

Reply:
`MAX_NODES` was meant to indicate the number of nodes running at the same time, IIRC.